interface/usergroup/user_admin.php

```diff
@@ -278,7 +286,12 @@ function authorized_clicked() {
     <TD style="width:180px;"><span class=text></span></TD>
     <TD style="width:270px;"></td>
     <TD style="width:200px;"><span class=text><?php echo xlt('User\'s New Password'); ?>: </span></TD>
-    <TD class='text' style="width:280px;">    <input type=text name=clearPass style="width:150px;"  class="form-control" value=""><font class="mandatory"></font></td>
+    <TD class='text' style="width:280px;">
+        <input type='password' id=clearPass name=clearPass style="width:150px;"  class="form-control" value="">
+        <input type="checkbox" id="togglePass" name="togglePass" onclick="toggle_password()" style="margin: .5rem 0 1rem;">
+        <label for="togglePass">Show Password</label>
+        <font class="mandatory"></font>
```

Comment:
hi @richcow2018 , looks like the `<font>` element is [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font).

Reply:
<font> is original there. We have not removed or added this line of code.

Reply:
good point :)